test-lit/lit.cfg.py

```diff
@@ -0,0 +1,68 @@
+# -*- Python -*-
+
+import os
+import platform
+import re
+import subprocess
+import sys
+
+import lit.formats
+import lit.util
+
+from lit.llvm import llvm_config
+from lit.llvm.subst import ToolSubst
+from lit.llvm.subst import FindTool
+
+# Configuration file for the 'lit' test runner.
+
+# name: The name of this test suite.
+config.name = 'Flang'
+
+# testFormat: The test format to use to interpret tests.
+#
+# For now we require '&&' between commands, until they get globally killed and
+# the test runner updated.
+config.test_format = lit.formats.ShTest(not llvm_config.use_lit_shell)
+
+
+# suffixes: A list of file extensions to treat as test files.
+config.suffixes = ['.f', '.F', '.FOR', '.for', '.f77', '.f90', '.F90',
```

Comment:
Does this list of suffixes need to match the list in lit.local.config?

Reply:
Should we have a set of tests that verify the suffix list?  Is there a way to check that a test file *isn't* recognized?

Reply:
So this patch has being rebase to this one:
https://github.com/flang-compiler/f18/pull/861